### PR TITLE
Added 'Crafted with Passion by DoDAO' to the footer

### PR DESCRIPTION
**Pull request by MSamiTariq**
Added 'Crafted with Passion by DoDAO' to the footer

### CODE REVIEW

File: src/components/home/TidbitsHub/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export default function Footer() {
+  return (
+    <footer>
```
Comment (RobinNagpal) [inline src/components/home/TidbitsHub/components/Footer.tsx:3]: We should add a link to DoDAO

File: src/components/home/TidbitsHub/TidbitsHubHome.tsx [hunk 2]
```diff
@@ -10,6 +11,7 @@ export default function TidbitsHubHome() {
       <PrimaryFeatures />
       <SecondaryFeatures />
       <Faqs />
+      <Footer />
```
Comment (RobinNagpal) [inline src/components/home/TidbitsHub/TidbitsHubHome.tsx:14]: This should be shown on all the sites, we can include it as part of `src/components/layout/BasePage.tsx`

File: src/components/home/TidbitsHub/components/Footer.tsx (hunk 1 above)
Comment (RobinNagpal) [reply src/components/home/TidbitsHub/components/Footer.tsx:3]: Add some margin below and the text can be made a bit smaller

**Comment by RobinNagpal** [comment]
PR for https://github.com/RobinNagpal/dodao-ui/issues/155

File: src/components/home/TidbitsHub/TidbitsHubHome.tsx (hunk 2 above)
Comment (MSamiTariq) [reply src/components/home/TidbitsHub/TidbitsHubHome.tsx]: Done

**Pull request merged**